pkg/gecko_sdk/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME=gecko_sdk
 PKG_URL=https://github.com/basilfx/RIOT-gecko-sdk
-PKG_VERSION=d381e526d68a2d0c951f37040c1c2e168ac66cd6
+PKG_VERSION=a5e7be18ef3d7947ab6f04f9cd74f8808da7aece
```

Comment:
Why do you need to update ?

Reply:
It includes an update for the I2C driver.